### PR TITLE
feat: added incident_summary view

### PR DESCRIPTION
**Pull request by adityathebe**
Resolves https://github.com/flanksource/duty/issues/105

### CODE REVIEW

**Comment by adityathebe** [comment]
```
-[ RECORD 1 ]-------------------------
id                  | 8305385a-0e76-490b-a5d2-e51c79bcfe25
title               | Broken
severity            | Low
type                | availability
status              | open
created_at          | 2023-03-21 12:09:37.193753
updated_at          | 2023-03-21 12:09:37.193753
commander           | {"id": "aab5b129-4195-4a65-b0d3-e0adf109d848", "name": "Aditya Thebe", "avatar": null}
distinct_responders | [{"id": "aab5b129-4195-4a65-b0d3-e0adf109d848", "name": "Aditya Thebe", "avatar": null}]
distinct_commenters | <null>
```

**Comment by moshloop** [comment]
@adityathebe I think just call them responders and commenters

**Comment by moshloop** [comment]
@adityathebe can we add a test for this query - if we can add dummy data quickly then we can verify the results but at a minimum we should verify that the views runs

File: fixtures/dummy/incidents.go [hunk 1]
```diff
@@ -24,32 +24,51 @@ var FirstComment = models.Comment{
...
 	UpdatedAt:  time.Now(),
 }
 
-var AllDummyComments = []models.Comment{FirstComment, SecondComment}
+var AllDummyComments = []models.Comment{FirstComment, SecondComment, ThirdComment}
 
 var FirstResponder = models.Responder{
 	ID:         uuid.New(),
 	IncidentID: LogisticsAPIDownIncident.ID,
 	Type:       "whattype",
```
Comment (yashmehrotra) [inline fixtures/dummy/incidents.go:54]: Can you change this to a Generic Type like `Jira` or `MSPlanner` ?

File: fixtures/dummy/incidents.go [hunk 2]
```diff
@@ -24,32 +24,51 @@ var FirstComment = models.Comment{
 	CreatedBy:  JohnWick.ID,
 	Comment:    "This is a comment",
 	IncidentID: LogisticsAPIDownIncident.ID,
-	CreatedAt:  time.Now(),
-	UpdatedAt:  time.Now(),
+	CreatedAt:  time.Now().Add(-time.Hour),
```
Comment (yashmehrotra) [inline fixtures/dummy/incidents.go:27]: Why are we doing this ?

Comment (adityathebe) [reply fixtures/dummy/incidents.go:27]: The test fails sometimes and I think it's because of the unstable ordering of the comments, Although I am not able to reproduce it enough times to make sure of it. I tried a few different ways to rewrite the SQL query to order the comment but the query was too verbose.

I was just experimenting with time to see if it helps in stable ordering.

Comment (adityathebe) [reply fixtures/dummy/incidents.go]: I have modified the query
